pkg/runs/runsDownload.go

```diff
@@ -227,20 +229,32 @@ func GetArtifactPathsFromRestApi(runId string, apiClient *galasaapi.APIClient) (
 	var artifactPaths []string
 	log.Println("Retrieving artifact paths for the given run")
 
-	var artifactsList []galasaapi.ArtifactIndexEntry
-	artifactsList, httpResponse, err := apiClient.ResultArchiveStoreAPIApi.
-		GetRasRunArtifactList(context.Background(), runId).
-		Execute()
+	var restApiVersion string
+	restApiVersion, err = embedded.GetGalasactlRestApiVersion()
 
 	if err != nil {
-		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_RETRIEVING_ARTIFACTS_FAILED, err.Error())
+		log.Printf("Unable to retrieve galasactl rest api version.")
```

Comment:
Shouldn't we have this as part of the parameter checker and just check if err !=nil before continuing. Lots of repetition of the same code here

Reply:
Made the error returning from the embedded function more readable to remove this everywhere.